.gitignore

```diff
@@ -1,2 +1,5 @@
 .vscode/
-./server/package-lock.json
\ No newline at end of file
+./server/package-lock.json
+./package-lock.json
+./node_modules
+./package.json
```

Comment:
don't add package.json in the .gitignore because it's the MAIN file that will tell us which dependencies you update. If you put it in gitignore and when you install something, and push to Git. Git will ignore the package file, and we don't know which dependencies you use. 
Now, it will not affect much, but I will remove it so no worry.